From 3ce2b97222bd612e3fec28fded4f8161d7d39394 Mon Sep 17 00:00:00 2001
From: bigrushdog <randall.rushing@gmail.com>
Date: Mon, 9 Jan 2017 01:55:29 -0800
Subject: [PATCH] Create "theme extras" directory

This creates /data/system/theme in init.rc. Historically, this
was done in ThemeService in CMTE. However, in a OMS/Subs
environment, OverlayManagerService is strictly dedicated to
handling overlays and nothing more.

Change-Id: I775e41310019695c4db82ccc9916dd14c8f690cd
---
 rootdir/init.rc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index c146ccf33..b76c0cc91 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -467,6 +467,7 @@ on post-fs-data
     mkdir /data/system 0775 system system
     mkdir /data/system/heapdump 0700 system system
     mkdir /data/system/users 0775 system system
+    mkdir /data/system/theme 0755 system system
 
     mkdir /data/system_de 0770 system system
     mkdir /data/system_ce 0770 system system
-- 
2.11.1

